### DISCUSSION TITLE
Posting the newest releases here on Github?

### DISCUSSION THREAD

**Discussion by jwsv61099** [microsoft/vscode-cpptools#9436] [General]
Hi all,

I noticed the dev team stopped posting about the new releases of this extension here on GitHub since the end of April, and I wondered if you guys were going to eventually continue posting again?

Thanks!

**Comment by sean-mcmanus** [comment]
@jwsv61099 Yeah, sorry -- we just forgot about it after publishing 1.10.5 to the marketplace: https://github.com/microsoft/vscode-cpptools/releases/tag/v1.10.5

**Reply by jwsv61099** [reply]
No worries! You guys are busy. Not a problem!
